src/Rule/IsTrue.php

```diff
@@ -57,31 +93,62 @@ public function getName(): string
         return 'isTrue';
     }
 
+    /**
+     * A getter for {@see $trueValue} property.
+     *
+     * @return scalar The value that is considered to be "true".
+     */
     public function getTrueValue(): int|float|string|bool
     {
         return $this->trueValue;
     }
 
+    /**
+     * A getter for {@see $strict} property.
+     *
+     * @return bool Whether the comparison to {@see $trueValue} is strict:
```

Comment:
```suggestion
     * @return bool Whether the comparison to {@see $trueValue} is strict.
```

Reply:
Closing as a duplicate of https://github.com/yiisoft/validator/pull/485#discussion_r1055633538 and I already replied there.

Reply:
What's the reason of this change? The colon is followed by a list, so I think it's OK.

Reply:
https://www.grammarbook.com/blog/colons/colons/